src/xmipp/libraries/reconstruction/reconstruct_art_pseudo.cpp

```diff
@@ -30,7 +30,7 @@
 #include "core/geometry.h"
 #include "core/xmipp_image.h"
 
-constexpr signed int FORWARD =   1;
+constexpr int FORWARD =   1;
 constexpr signed int BACKWARD = -1;
```

Comment:
in C++, `int` is always signed, so `signed` keyword is almost always unnecessary

Reply:
thanks